plugin/trino-hive/src/main/java/io/trino/plugin/hive/parquet/ParquetPageSourceFactory.java

```diff
@@ -332,6 +348,31 @@ public static ReaderPageSource createPageSource(
         return Optional.of(new GroupType(baseType.getRepetition(), baseType.getName(), ImmutableList.of(type)));
     }
 
+    private static ColumnIndexStore getColumnIndexStore(Predicate parquetPredicate, ParquetDataSource dataSource, BlockMetaData blockMetadata, Map<List<String>, RichColumnDescriptor> descriptorsByPath, boolean readColumnIndexFilter)
```

Comment:
I'd name the third parameter just `block`, and wrap these parameters onto their own lines.

Once `readColumnIndexFilter` is part of `ParquetReaderOptions`, remove that parameter, make this method non-static, and use `this.options.getReadColumnIndexFilter()` instead.